assets/src/dashboard/components/navigationBar.js

```diff
@@ -135,9 +133,9 @@ function NavigationBar() {
         ))}
       </PageLinks>
 
-      <NewStoryLink forwardedAs="a" type={BUTTON_TYPES.CTA} href={newStoryURL}>
+      <Button type={BUTTON_TYPES.CTA} href={newStoryURL} isLink={true}>
```

Comment:
Can a legit `button` element (not an anchor that looks like button) have an `href`?  If not, could we infer that it's a button if `href=undefined` and infer it's an anchor if `href` has a value?  And then we wouldn't need to pass in `isLink`.  Would that work or am I missing a use case? 🤔

Reply:
i did that at first, but decided to separate it because, even though it is redundant to have both it's clearer when you read it and it allows `onClick` vs `href` to be passed as part of `...rest` and made the actual Button component more presentational as it's just deciding what kind of element to display instead of also setting what happens in it.  

if you feel like it's more worthwhile to rely on just href to set button vs anchor and then we'd set href={href} outside of the ...rest i can update it. 

To me, this was cleaner.

Reply:
I'm good with clearer, so looks good to me! 👍